libs/community/langchain_community/chat_models/maritalk.py

```diff
@@ -1,10 +1,25 @@
+import json
 from http import HTTPStatus
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, AsyncIterator, Dict, List, Optional, Union
 
+import httpx
```

Comment:
Since `httpx` is an [optional dependency](https://github.com/langchain-ai/langchain/blob/40f846e65da37a1c00d72da9ea64ebb0f295b016/libs/community/pyproject.toml#L92) of langchain-community, we should ensure users can import this code without the dependency installed with no side effects. See [contributing guide](https://python.langchain.com/docs/contributing/code#working-with-optional-dependencies). Consider moving this import inside the class (e.g., as is done [here](https://github.com/langchain-ai/langchain/blob/40f846e65da37a1c00d72da9ea64ebb0f295b016/libs/community/langchain_community/llms/chatglm3.py#L76)).